dime-research-standards/pillar-1-research-ethics/README.md

```diff
@@ -20,7 +20,7 @@
 
 ## Human Subjects Research Certification
 **All research team members that handle personally identifiable information must have up to date Human Subjects Research Certification.**
-- DIME-affiliated research team members may obtain the certification through DIME's [PHRP](https://phrptraining.com/) subscription. To do so, please email DIME Analytics.
+- DIME-affiliated research team members may obtain the certification through DIME's [CITI](https://duckduckgo.com/?t=ffab&q=citi+human+subjects+training&ia=web) subscription. To do so, please email DIME Analytics.
```

Comment:
is it limited to DIME affiliates? i thought once we put on eservices and added a cost it was open to pretty much any one at the Bank. and doesn't work for DIME affiliates outside the Bank, right?

Reply:
this comment is no longer applicable as the process for CITI certification request has changed. the updated request instructions are included

Reply:
@roshni13khincha just realized that the updated link goes to the search result for CITI's training instead of CITI's site. Is that intended?